packages/relay/src/lib/types/IMetricService.ts

```diff
@@ -0,0 +1,25 @@
+/* -
```

Comment:
`IMetricService` suggests that this file contains the interface of the `MetricService` class and I don't see it here, this name is very misleading.

Rename this file to `transactionRecordMetric.ts` if it's going to contain only one interface, or `metrics.ts` to suggest that this could contain more than one metric type.

Reply:
Yeah good point! Renamed to `metrics.ts`